docs/en/contributor/Contributor-Tools.md

```diff
@@ -51,6 +51,29 @@ And then check the box next to "developer" in the window pane.
 > Note: Alluxio 2.2 moved generated gRPC proto source files into `alluxio/core/transport/target/generated-sources/protobuf/`.
 You will need to mark the directory as "Generated Sources Root" for IntelliJ to resolve the source files.
 
+#### Run in IntelliJ IDEA
+1. Run `dev/intellij/install-runconfig.sh`
+2. Restart IntelliJ IDEA
+3. Edit conf/alluxio-site.properties
+```console
+  alluxio.home={alluxio.home}
+  alluxio.master.hostname=localhost
+  alluxio.master.journal.type=UFS
+```
+4. Edit conf/log4j.properties to print log in console
+```console
+  log4j.rootLogger=INFO, ${alluxio.logger.type}, ${alluxio.remote.logger.type}, stdout
+  log4j.threshold=ALL
+  log4j.appender.stdout=org.apache.log4j.ConsoleAppender
+  log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
+  log4j.appender.stdout.layout.ConversionPattern=%d{ISO8601} %-5p %c{2} (%F:%M(%L)) - %m%n
+```
+5. bin/alluxio formatMaster
+6. `Run > Run > AlluxioMaster`
+7. bin/alluxio-mount.sh SudoMount && bin/alluxio formatWorker
+8. `Run > Run > AlluxioWoker`
```

Comment:
Can this be something like 
`In Intellij, start Alluxio worker process by selecting Run > Run > AlluxioWorker`